### ISSUE TITLE
Please update older versions with getmypid alternative

### ISSUE THREAD

**Issue by Cipa** [swiftmailer/swiftmailer#1025]
https://github.com/swiftmailer/swiftmailer/blob/412333372fb6c8ffb65496a2bbd7321af75733fc/lib/classes/Swift/Mime/SimpleMimeEntity.php#L816

Older projects should be updated with bin2hex(random_bytes(16)) instead of md5(getmypid().mt_rand().uniqid('', true))

I have a project on php 5.6 and swiftmailer won't go higher than 5.4. Because of getmypid is not allowed on the server I have to manually update those lines with bin2hex(random_bytes(16)).

Here's my composer file

```
{
    "name": "some/project",
    "config": {
        "platform": {
            "php": "5.6.32"
        }
    },
    "require": {
        "plasticbrain/php-flash-messages": "^1.0",
        "illuminate/database": "^5.4",
        "wilber/wilbercsv": "^1.0",
        "mandrill/mandrill": "^1.0",
        "swiftmailer/swiftmailer": "^5.4"
    }
}
```

Thank you

**Comment by Rotzbua** [comment]
You are funny, `random_bytes` is introduced in php7. https://secure.php.net/manual/de/function.random-bytes.php

**Comment by Cipa** [comment]
well it says

Note: Although this function was added to PHP in PHP 7.0, a » userland implementation is available for PHP 5.2 to 5.6, inclusive.

Seems to work for me, better than getmypid

**Comment by Rotzbua** [comment]
"userland implementation" means extra implementation, means extra library, means extra work.
If your integration works for you, then make a pull request. Cheers.
